core/vm/contracts.go

```diff
@@ -223,7 +227,7 @@ func ActivePrecompiles(rules params.Rules) []common.Address {
 		return PrecompiledAddressesIstanbul
 	case rules.IsByzantium:
 		return PrecompiledAddressesByzantium
-	default:
+	default: // Any ArbOS-disabled node
```

Comment:
Isn't this a misleading comment?
Everything except PrecompiledAddressesStartingFromArbOS50, PrecompiledAddressesAfterArbOS30 and PrecompiledAddressesBeforeArbOS30 are not related to "ArbOS-disabled node"?